portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,128 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
-}
+* {box-sizing:border-box}
+  
+  #body {
+     background-image: linear-gradient(to right,rgba(64, 95, 151, 0.8), rgba(255,0,0,0), rgba(64, 95, 151, 0.8));
+  }
+  #content {
+    border-radius: 10px;
+    border: 5px solid #73AD21;
+    margin-left: auto;
+    margin-right: auto;
+    width: 90%;
+    background-color: rgba(255, 255, 255, 1);
+  }
 
-#greeting-container {
-  margin-top: 20px;
-}
+  #subcontent {
+    margin-left: auto;
+    margin-right: auto;
+    width: 95%;
+    background-color: rgba(255, 255, 255, 0);
+  }
+
+/*github, and linkedin buttons*/
+.button {
+    background-color: rgb(33, 9, 252); /* Royal Blue */
+    border-radius: 10px;
+    border: 2px solid rgb(139, 226, 18); /* Green */
+    color: white;
+    padding: 0px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+  }
+  
+  /* Slideshow container */
+  .slideshow-container {
+    max-width: 1200px;
+    width: 90%;
+    position: relative;
+    margin: auto;
+  }
+  
+  /* Hide the images by default */
+  .mySlides {
+    display: none;
+  }
+  
+  /* Next & previous buttons */
+  .prev, .next {
+    cursor: pointer;
+    position: absolute;
+    top: 50%;
+    width: auto;
+    margin-top: -22px;
+    padding: 16px;
+    color: white;
+    font-weight: bold;
+    font-size: 18px;
+    transition: 0.6s ease;
+    border-radius: 0 3px 3px 0;
+    user-select: none;
+  }
+  
+  /* Position the "next button" to the right */
+  .next {
+    right: 0;
+    border-radius: 3px 0 0 3px;
+  }
+  
+  /* On hover, add a black background color with a little bit see-through */
+  .prev:hover, .next:hover {
+    background-color: rgba(0,0,0,0.8);
+  }
+  
+  /* Caption text */
+  .text {
+    color: #f2f2f2;
+    font-size: 18px;
+    padding: 8px 12px;
+    position: absolute;
+    bottom: 1px;
+    width: 100%;
+    text-align: center;
+  }
+  
+  /* Number text (1/3 etc) */
+  .numbertext {
+    color: #f2f2f2;
+    font-size: 16px;
+    padding: 8px 12px;
+    position: absolute;
+    top: 0;
+  }
+  
+  /* The dots/bullets/indicators */
+  .dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+    transition: background-color 0.6s ease;
+  }
+  
+  .active, .dot:hover {
+    background-color: #717171;
+  }
+  
+  /* Fading animation */
+  .fade {
+    -webkit-animation-name: fade;
+    -webkit-animation-duration: 1.5s;
+    animation-name: fade;
+    animation-duration: 1.5s;
+  }
+  
+  @-webkit-keyframes fade {
+    from {opacity: .4}
+    to {opacity: 1}
+  }
+  
+  @keyframes fade {
+    from {opacity: .4}
+    to {opacity: 1}
+  }
+  
```

Comment:
I'm not quite sure how you ended up with an empty line and the "no newline" symbol in this file. I think it's because your last line has some spaces in it but I'm not 100% certain.